app/src/main/java/org/p2p/wallet/send/ui/main/SendPresenter.kt

```diff
@@ -695,4 +715,7 @@ class SendPresenter(
 
     private fun isAddressValid(address: String?): Boolean =
         !address.isNullOrBlank() && address.trim().length >= VALID_ADDRESS_LENGTH
+
+    private val Token.Active?.isRenBTC: Boolean
```

Comment:
Token.Active is a sealed class, so i think we should use some additional state for this class like None, or Empty to avoid using nullable

Reply:
It's a good point for future refactoring